tests/api_tests.py

```diff
@@ -93,6 +93,19 @@ def test_infrared_sensor(self):
         self.assertEqual(s.num_values,      1)
         self.assertEqual(s.address,         'in1')
         self.assertEqual(s.value(0),        16)
+        self.assertEqual(s.mode,            "IR-PROX")
+
+        s.mode = "IR-REMOTE"
+        self.assertEqual(s.mode,            "IR-REMOTE")
+
+        val = s.proximity
+        # Our test environment writes to actual files on disk, so while "seek(0) write(...)" works on the real device, it leaves trailing characters from previous writes in tests. "s.mode" returns "IR-PROXTE" here.
```

Comment:
There isn't an obvious easy fix here so I'm inclined to just leave this as-is; open to suggestions.